tests/lammps/test_base.py

```diff
@@ -844,6 +844,25 @@ def test_bonds_input(self):
         bond_str = "2 bond types\n"
         self.assertTrue(self.job["structure.inp"][4][-1], bond_str)
 
+    def test_multistep_log_parse(self):
+        structure = Atoms(
+            symbols=["Cu", "Cu", "Cu", "Cu"],
+            positions=[
+                [0.0, 0.0, 0.0],
+                [0.0, 1.805, 1.805],
+                [1.805, 0.0, 1.805],
+                [1.805, 1.805, 0.0],
+            ],
+            cell=[3.61, 3.61, 3.61],
+        )
+        file_directory = os.path.join(
+            self.execution_path, "..", "static", "lammps_test_files"
+        )
+        _generic_keys_lst, _pressure_dict, _df = _collect_output_log(
+            file_name=os.path.join(file_directory, "multi_log_step_test.lammps"),
+            prism=UnfoldingPrism(structure.cell),
+        )
+
```

Comment:
Can you validate the output here? For example you could check the length of the two individual runs and that all properties are parsed.